assets/css/main.css

```diff
@@ -3447,25 +3410,6 @@
         padding: 2.5em 0;
         text-align: center;
     }
-
-        #footer .icons .icon.alt {
```

Comment:
No reason to touch the footer icons, this should all be reverted as well.

Reply:
What is it used for?

Reply:
Any icons being placed in the footer area, was planning on using this for materialize. I've found a simpler way to do this since by embedding a .js file that can automatically fix it to screen size.